gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Mon Oct 14 12:32:29 PDT 2019
+#Thu Sep 01 12:16:51 MSK 2022
 distributionBase=GRADLE_USER_HOME
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.3.3-bin.zip
```

Comment:
By the way, the latest Gradle version is 7.5.1.

Reply:
Thanks, I updated it.